src/EntityFramework.SqlServer/SqlProviderServices.cs

```diff
@@ -132,6 +133,15 @@ public static bool TruncateDecimalsToScale
         }
 
         /// <summary>
+        /// Set this flag to false to use an OUTPUT clause instead of the SCOPE_IDENTITY() function to 
+        /// retrieve values generated on the database for numeric columns during an INSERT operation. 
+        /// This makes it possible to retrieve values generated by sequences or other means, but setting 
+        /// the flag to the default value of true is recommended and can provide better performance if all 
+        /// values are generated using IDENTITY columns.
+        /// </summary>
```

Comment:
This would align better with how we've written docs for other flags...

> Gets or sets a value indicating whether to use the SCOPE_IDENTITY() function retrieve values generated by the database for numeric columns during an INSERT operation. The default value of <c>true</c> is recommended and can provide better performance if all numeric values are generated using IDENTITY columns. If set to <c>false</c>, an OUTPUT clause will be used instead. An OUTPUT clauses makes it possible to retrieve values generated by sequences, or other means.